.erda/migrations/dop/20220801-devflow.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE `erda_dev_flow`
+(
+    `id`                      VARCHAR(36)  NOT NULL DEFAULT '' COMMENT 'id',
+    `org_id`                  BIGINT ( 20 ) NOT NULL DEFAULT 0 COMMENT 'orgID',
+    `org_name`                VARCHAR(50)  NOT NULL DEFAULT '' COMMENT 'orgName',
+    `app_id`                  BIGINT ( 20 ) NOT NULL DEFAULT 0 COMMENT 'projectID',
+    `app_name`                VARCHAR(50)  NOT NULL DEFAULT '' COMMENT 'projectName',
+    `creator`                 VARCHAR(36)  NOT NULL DEFAULT '' COMMENT 'creator',
+    `created_at`              datetime     NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'CREATED AT',
+    `updated_at`              datetime     NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'UPDATED AT',
+    `deleted_at`              BIGINT ( 20 ) NOT NULL DEFAULT '0' COMMENT 'deleted_at',
+    `branch`                  VARCHAR(50)  NOT NULL DEFAULT '' COMMENT 'branch',
+    `flow_rule_name`          VARCHAR(50)  NOT NULL DEFAULT '' COMMENT 'flowRuleName',
```

Comment:
what is `flow_rule_name`? why not use `rule_id` if have?

Reply:
flows are json without id